akka-http-core/src/main/scala/akka/http/impl/util/StreamUtils.scala

```diff
@@ -224,38 +226,34 @@ private[http] object StreamUtils {
    * is finished, only that the part that contains this flow has finished work.
    */
   def identityFinishReporter[T]: Flow[T, T, Future[Unit]] = {
```

Comment:
Two good pieces of news here:
- It's in internal api, we could change if it we want to
- we want to remove the entire StreamUtils object (which contains this), since those were workarounds for missing Streams features – so the goal is to remove these workaround-thingies all together